NOTICE.txt

```diff
@@ -1,16 +1,20 @@
 ==============================================================
  JanusGraph: Distributed Graph Database
- Copyright 2012 and onwards Aurelius
+ Copyright 2012 and onwards JanusGraph Authors
 ==============================================================
 
-This product includes software developed by Aurelius (http://thinkaurelius.com/) and the following individuals:
+This product includes software developed by JanusGraph contributors; the list
+can be found in CONTRIBUTORS.txt; the list of JanusGraph copyright holders can
+be found in AUTHORS.txt .
+
```

Comment:
Sure, since you insist. I've also simplified the wording and shortened the sentence.